plugins/ledger/ledger.py

```diff
@@ -254,7 +265,8 @@ def wrapper(self, *args, **kwargs):
     def cashaddr_alert(self):
         """Alert users about fw/sw updates for cashaddr."""
         if Address.FMT_UI == Address.FMT_CASHADDR:
-            if not self.get_client_electrum().fw_supports_cashaddr():
+            # Do not warn if the device is HW1, they have no display anyway
+            if not self.get_client_electrum().fw_supports_cashaddr() and not self.get_client_electrum().is_hw1():
                 self.handler.show_warning(MSG_NEEDS_FW_UPDATE_CASHADDR)
```

Comment:
We maybe should display some kind of warning to the user.
It is not possible to verify the address any more as the address is displayed in base58 during the second factor authentication.

Reply:
Yea, it would make sense to display some warning. It would also be good to be able to disable the warning (don't show again checkbox or a setting).

Reply:
This is resolved now. Both addresses are displayed when cashaddresses are enabled.